src/commands/react-native/interfaces.ts

```diff
@@ -0,0 +1,80 @@
+import fs from 'fs'
+
+import {MultipartPayload, MultipartValue} from '../../helpers/upload'
+
+export class RNSourcemap {
+  public bundleName: string
+  public bundlePath: string
+  public gitData?: GitData
+  public sourcemapPath: string
+
+  constructor(bundlePath: string, sourcemapPath: string, bundleName?: string) {
+    this.bundleName = this.getBundleName(bundlePath, bundleName)
+    this.bundlePath = bundlePath
+    this.sourcemapPath = sourcemapPath
+  }
+
+  public addRepositoryData(gitData: GitData) {
+    this.gitData = gitData
+  }
+
+  public asMultipartPayload(
+    cliVersion: string,
+    service: string,
+    version: string,
+    projectPath: string,
+    platform: RNPlatform,
+    build: string
+  ): MultipartPayload {
+    const content = new Map<string, MultipartValue>([
+      ['cli_version', {value: cliVersion}],
+      ['service', {value: service}],
+      ['version', {value: version}],
+      ['build_number', {value: build}],
```

Comment:
I don't think the backend is expecting any `build_number`, but `bundle_version` instead cc @rpelliard